lib/geoblacklight/constants.rb

```diff
@@ -14,7 +14,11 @@ module Constants
       wcs: 'http://www.opengis.net/def/serviceType/ogc/wcs',
       wfs: 'http://www.opengis.net/def/serviceType/ogc/wfs',
       wms: 'http://www.opengis.net/def/serviceType/ogc/wms',
-      hgl: 'http://schema.org/DownloadAction'
+      hgl: 'http://schema.org/DownloadAction',
+      feature_layer: 'http://www.arcgis.com/rdf#FeatureLayer',
```

Comment:
Can you remind me why we decided on RDF here? To me referencing "RDF" feels like it should actually be rdf, no?